app/assets/stylesheets/components/_youtube-player.scss

```diff
@@ -40,28 +41,31 @@
   }
 }
 
-.govuk-details__summary {
-  color: govuk-colour('white');
-  &:hover {
+.youtube__video-wrapper--inverse {
+  .govuk-details__summary {
     color: govuk-colour('white');
+    &:hover {
```

Comment:
Nitpick: I'd put an empty line before the `&:hover`, and similarly for the `&:focus` below.